app/views/user_mailer/notify_collection_failed.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+  <body>
+    <div style="background-color:LightGray; width: 100%; margin: auto;">
+      <p style="text-align: center;">
+        <%= image_tag attachments['AUK-Logo-full.png'].url, alt: 'Archives Unleashed Logo', style: 'display: inline-block; margin-left: auto; margin-right: auto; padding-top: 30px; padding-bottom: 30px;' -%>
+      </p>
+    </div>
+    <p>Hi <%= @user.auk_name %>,</p>
+    <p>Your Archives Unleashed Cloud job has failed. We are looking into it, and we'll let you know how to proceed soon.
```

Comment:
I think this works - it does raise the expectation for us to get back to them relatively quickly. But I think given the state of the Cloud right now this is reasonable.